deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -10,6 +10,7 @@ use serde::Serialize;
 use typescript_type_def::TypeDef;
 
 use super::contact::ContactObject;
+use super::webxdc::WebxdcMessageInfo;
```

Comment:
wondering, btw, why we canning use WebxdcInfo from core directly - these are the same fields - is it only because of the option-wrapping? or is there some magic going on that requires the structure to be in the rpc folder?

Reply:
typescript type generation macro, we could probably make a feature for core enabling the derive macro for generating typings on the type in core, but that complicates core code a bit, so for now this is easier even if it might have a small overhead.

Reply:
thanks for explanation, makes totally sense then.